Appp.py

```diff
@@ -0,0 +1,27 @@
+from board import *
+class App(tk.Tk):
+    def __init__(self, *args, **kwargs):
+        tk.Tk.__init__(self, *args, **kwargs)
+        self.canvas = tk.Canvas(self, width=640, height=640, borderwidth=0, highlightthickness=0)
+        self.canvas.pack(side="top", fill="both", expand="true")
+        self.Game = Game()
+        self.rows = 8
+        self.columns = 8
```

Comment:
The board is fixed to 8x8, also the logic is developed as it. So you don't really need these lines